addon/routes/gis-search-form.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { Query } from 'ember-flexberry-data';
+
+export default Ember.Route.extend({
+  _metadatas: [],
+  _maps: [],
+
+  model() {
+    return { metadata: this._metadatas, maps: this._maps };
+  },
+
+  //setupController
+  actions: {
+    loadData(keyWords) {
```

Comment:
really, it is better to make this using model() hook and queryParams with reloadModel: true attribute.